apps/ui/src/views/Editor.vue

```diff
@@ -467,7 +543,24 @@ export default defineComponent({
           enforcedVoteType ? [enforcedVoteType] : space.voting_types
         "
       />
-      <EditorChoices v-model="proposal" :definition="CHOICES_DEFINITION" />
+      <EditorChoices
+        v-model="proposal"
+        :definition="choicesDefinition"
+        :error="
+          proposal.choices.length > choicesDefinition.maxItems
+            ? `Must not have more than ${_n(choicesDefinition.maxItems)} items.`
+            : ''
+        "
+      >
+        <template v-if="!space?.turbo && isOffchainSpace" #error-suffix>
```

Comment:
Btw, I'm not sure to understand why error and error suffix are using 2 different params, this could just be a single "error" template slot, that would give more freedom on how we format errors message.

Reply:
> Btw, I'm not sure to understand why error and error suffix are using 2 different params, this could just be a single "error" template slot, that would give more freedom on how we format errors message.

This follows a similar approach to the `Composer` component above, which uses `error` prop to indicate error on the border, limit text etc.
<img width="693" alt="image" src="https://github.com/user-attachments/assets/e1f2e89e-d03b-45a6-ada3-2d0b1db28e84"> 

Also Composer is used on profile statement as well, where we don't need the suffix, this approach will not disturb that, without having to change the statement component. 

For EditorChoices, this will be useful  if we want to show other errors/indications in the future

Reply:
i can merge this PR?